ci/default.nix

```diff
@@ -117,6 +117,14 @@ let
           command = "${pkgs.lib.getExe pkgs.editorconfig-checker}";
           options = [ "-disable-indent-size" ];
           includes = [ "*" ];
+          excludes = [
+            # These are binary files. Without this next exclude pattern, they
+            # would mistakenly be interpreted as text files, and
+            # editorconfig-checker would complain that it can’t detect the
+            # character encoding for some of them and that most of them don’t
+            # end with a newline.
+            "lib/byte-to-hex-table/??"
+          ];
```

Comment:
You should be able to create a local `.editorconfig` file in the relevant folder to tell the checker that these things are binary files. No need for an exclusion here, imho.